Python/Product/Cookiecutter/Model/CookiecutterClient.cs

```diff
@@ -141,6 +147,26 @@ class CookiecutterClient : ICookiecutterClient {
             MoveToDesiredFolder(outputFolderPath, tempFolder);
         }
 
+        public Task<string> GetDefaultOutputFolderAsync(string shortName) {
```

Comment:
It'll have to become some sort of query event then, or a new object that gets passed into the view model, as it doesn't make sense to have this function anywhere else in the current object model. (The problem being that users can change their project path while VS is running - if it were static then it wouldn't matter.)